namespaces/live-1.cloud-platform.service.justice.gov.uk/contact-moj-staging/resources/rds.tf

```diff
@@ -0,0 +1,55 @@
+################################################################################
+# contact-moj
+# Application RDS (PostgreSQL)
+#################################################################################
+
+module "contact-moj_rds" {
+  source                     = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.1"
```

Comment:
Can you use the latest module(5.3) please

Reply:
Done, thanks